lib/internal/util/inspect.js

```diff
@@ -1530,8 +1530,6 @@ function formatPrimitive(fn, value, ctx) {
       trailer = `... ${remaining} more character${remaining > 1 ? 's' : ''}`;
     }
     if (ctx.compact !== true &&
-        // TODO(BridgeAR): Add unicode support. Use the readline getStringWidth
-        // function.
```

Comment:
Isn't it better to update the comment and clarify that unicode is not supported on purpose for performance reasons?